style.css

```diff
@@ -0,0 +1,84 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.home-page {
+  background-color: #fff;
+  border-radius: 0 0 0 5%;
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+}
+
+.log {
+  height: 30px;
+  width: 30px;
+  display: flex;
+  align-items: center;
+}
+
+.hamburger {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem;
+}
+
+#home {
+  background-image: url("./img/imagee.png");
+  background-size: 25%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin: 8px 12px 0 12px;
+  padding-top: 25px;
+  padding-bottom: 48px;
```

Comment:
**OPTIONAL** - According to the project requirements, this section should be aligned with the `flex` display property because it gives us a lot of options to align all our elements, plus, it's responsive. Please fix this 👍 👍